Editor/CreatorProjectSettings.cs

```diff
@@ -13,6 +13,18 @@ public class CreatorProjectSettings : ScriptableObject
     [SerializeField]
     public bool IsFirstTimeStarted = true;
 
+    /// <summary>
+    /// Enables the validation system.
+    /// </summary>
+    [SerializeField]
+    public bool IsValidationEnabled = true;
+
+    /// <summary>
+    /// Enables the validation system debugging output.
+    /// </summary>
+    [SerializeField]
+    public bool DebugValidationSystem = false;
```

Comment:
Should this be in the core? It is pretty useless without the validation thingi.

Reply:
Either under a "Pro" section or only visible if you have Pro?

Reply:
Made the class partial, which allows different components to contribute settings, but I'm not sure this is a good way. If stuff is missing and it's changed/saved parts of settings are lost.